README.md

```diff
@@ -1212,7 +1220,11 @@ For more, see [Pylint](https://pypi.org/project/pylint/) on PyPI.
 | PLR0402 | consider-using-from-import | Use `from {module} import {name}` in lieu of alias |  |
 | PLR1701 | consider-merging-isinstance | Merge these isinstance calls: `isinstance({obj}, ({types}))` |  |
 | PLR1722 | use-sys-exit | Use `sys.exit()` instead of `{name}` | 🛠 |
-| PLR2004 | magic-value-comparison | Magic value used in comparison, consider replacing {value} with a constant variable |  |
+
+
+The following rules have been disabled by default:
+
+* PLR2004 (magic-value-comparison): Comparisons to strings are generally not considered to be an anti-pattern.
```

Comment:
We ignore comparisons to strings by default. At least, that's the intent (it's parameterizable). If so, this message is incorrect, I think?

Reply:
Ah ... I hadn't noticed the `allow-magic-value-types` setting you introduced in #1987, which even defaults to `["str"]`. Ok yes this addresses the problem.

Reply:
I have dropped the default disabling of MagicValueComparison from this PR.